docs/_src/tutorials/tutorials/2.md

```diff
@@ -92,6 +92,40 @@ reader.save(directory="my_model")
 new_reader = FARMReader(model_name_or_path="my_model")
 ```
 
+## Distill your model
+In this case, we have used "distilbert-base-uncased" as our base model. This model was trained using a process called distillation. In this process, a bigger model is trained first and is used to train a smaller model which increases its accuracy. This is why "distilbert-base-uncased" can achieve quite competitive performance while being very small.
+
+Sometimes, however, you can't use an already distilled model and have to distil it yourself. For this case, haystack has implemented a [distillation approach](https://arxiv.org/pdf/1909.10351.pdf).
```

Comment:
Let's link to our guide here instead: https://haystack.deepset.ai/guides/model-distillation The guide then includes the link to the paper.